support/runtime-upgrade/README.md

```diff
@@ -0,0 +1,3 @@
+# Simple JS tool for substrate upgrades
+
+Eaxmple: `./support/runtime-upgrade/upgrade_chain.sh ws://127.0.0.1:9944 ./support/runtime-upgrade/example_mnemonic`
```

Comment:
Don't include the directory the script is in for examples like this. It adds unnecessary noise and it would be incorrect if the file is moved. Consumers can figure out where to run it from.